examples/sglang/utils/wideep/sgl_bench_one_batch_server.py

```
@@ -0,0 +1,380 @@
+    use_structured_outputs = False
+    if use_structured_outputs:
+        texts = []
+        for _ in range(batch_size):
+            texts.append(
+                "Human: What is the capital city of france? can you give as many trivial information as possible about that city? answer in json.\n"
+                * 50
+                + "Assistant:"
+            )
+        json_schema = "$$ANY$$"
+    else:
+        json_schema = None
```

Flake8 (7.2.0)

[error] local variable 'json_schema' is assigned to but never used

(F841)